servers/rendering/renderer_scene_render.cpp

```diff
@@ -29,3 +29,153 @@
 /*************************************************************************/
 
 #include "renderer_scene_render.h"
+
+void RendererSceneRender::ViewData::set_single_view(const Transform3D p_transform, const CameraMatrix p_projection, bool p_is_ortogonal, bool p_vaspect) {
```

Comment:
I would probably also rename those into set_camera and set_multiview_camera, to make sure multiview is the exception.

Reply:
Sounds good to me, I'll make it so